packages/trader/src/Modules/MT5/Components/mt5-demo-account-display.jsx

```diff
@@ -87,12 +87,7 @@ const MT5DemoAccountDisplay = ({
                         type: 'financial',
                     }}
                     existing_data={current_list['demo.financial']}
-                    commission_message={
-                        <Localize
-                            i18n_default_text='No commission <0>(excluding cryptocurrencies)</0>'
-                            components={[<span key={0} className='mt5-dashboard--hint' />]}
-                        />
-                    }
+                    commission_message={<Localize i18n_default_text='No commission' />}
```

Comment:
Now that the prop has no internal HTML, can we simply use `localize` instead of `<Localize />`? (same at line 131 and the `mt5-real-account-display.jsx`)